### PR TITLE
refactor: remove unused deps

### PR DESCRIPTION
**Pull request by EYHN**
auto removed by https://knip.dev/features/auto-fix

### CODE REVIEW

**Comment by EYHN** [comment]
* **#8259** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8259?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#8258** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8258?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/native/package.json [hunk 1]
```diff
@@ -36,7 +36,6 @@
     "@napi-rs/cli": "3.0.0-alpha.62",
     "lib0": "^0.2.93",
     "nx": "^19.0.0",
-    "nx-cloud": "^19.0.0",
```
Comment (darkskygit) [inline packages/backend/native/package.json]: this package is necessary for reuse cloud cache

File: packages/backend/server/package.json [hunk 2]
```diff
@@ -81,30 +77,22 @@
...
     "openai": "^4.33.0",
-    "parse-duration": "^1.1.0",
     "piscina": "^4.5.1",
-    "pretty-time": "^1.1.0",
     "prisma": "^5.12.1",
-    "prom-client": "^15.1.1",
     "reflect-metadata": "^0.2.2",
     "rxjs": "^7.8.1",
-    "semver": "^7.6.0",
-    "ses": "^1.4.1",
```
Comment (darkskygit) [inline packages/backend/server/package.json:92]: used in copilot's workflow

File: packages/frontend/graphql/src/schema.ts [hunk 3]
```diff
@@ -1,41 +1,28 @@
 /* eslint-disable */
```
Comment (darkskygit) [inline packages/frontend/graphql/src/schema.ts]: this file is automatically generated and should not be formatted

File: packages/frontend/apps/web/package.json [hunk 4]
```diff
@@ -12,18 +12,14 @@
   "dependencies": {
     "@affine/component": "workspace:*",
     "@affine/core": "workspace:*",
-    "@affine/electron-api": "workspace:*",
-    "@affine/env": "workspace:*",
     "@affine/i18n": "workspace:*",
     "@emotion/react": "^11.11.4",
     "@sentry/react": "^8.0.0",
-    "jotai": "^2.8.0",
```
Comment (pengx17) [inline packages/frontend/apps/web/package.json:20]: better keep it for type language service's auto-complete in vscode

**Review by darkskygit — APPROVED** [review]

**Pull request merged**